adapters/pywasm-runtime.py

```diff
@@ -0,0 +1,28 @@
+import argparse
```

Comment:
Thanks, would it be possible to keep the file in the pywasm repository? See wasmedge for example.

Reply:
Okay, that's a good idea indeed.